Projekty/Projekt2/Marcin Wilk, Hubert Drążkowski/[Grupa_4][Milestone_1]WilkDrazkowski.ipynb

```diff
@@ -0,0 +1,1266 @@
+{
```

Comment:
<p>Tutaj zgadzam się w 100%. Nie wiem czy na prezentacji wystarczający wyraźnie to powiedziałem.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/317/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>